tox.ini

```diff
@@ -17,7 +17,7 @@ envlist = py37,py38,lint,type
 
 [testenv]
 deps = .[test]
-commands = pytest
+commands = pytest {posargs}
```

Comment:
Can you explain why this was added? I don't understand what it does.

Reply:
this allows to test a specific file / test fixture

Reply:
That seems useful! I'm ok with having it in this change. It can also be in its own CL.